sdks/python/README.md

```diff
@@ -16,6 +16,7 @@
     specific language governing permissions and limitations
     under the License.
 -->
+== This page is currently being updated. ==
```

Comment:
== was not meant to render, I just liked the view.
In the next commit, I'll explain that all the examples have been updated, but other references to alpha have not. Cyril just told me that he filed a bug for Eric to take care of that.